ui/components/app/connected-status-indicator/index.scss

```
@@ -8,11 +8,11 @@
   border-radius: 100px;
 
   &:hover {
-    background-color: #f2f3f4;
+    background-color: var(--color-background-alternative);
   }
 
   &:active {
-    background-color: #ededed;
+    background-color: var(--color-background-default);
```

Can just leave it the same as hover we don't have a color for this. Really it should be using the secondary type Button component

```suggestion
    background-color: var(--color-background-alternative);
```